grid2op/MakeEnv/Make.py

```diff
@@ -159,9 +155,10 @@ def _extract_ds_name(dataset_path):
     dataset_name = os.path.splitext(dataset_name)[0]
     return dataset_name
 
-def _aux_is_multimix(dataset):
-    ds_name = os.path.basename(dataset)
-    if ds_name in MULTIMIX_ENVS:
+_MULTIMIX_FILE = ".multimix"
+
+def _aux_is_multimix(dataset_path):
+    if os.path.exists(os.path.join(dataset_path, _MULTIMIX_FILE)):
         return True
     return False
```

Comment:
`return os.path.exists(os.path.join(dataset_path, _MULTIMIX_FILE))` would probably be less confusing, but this works too ;-)